.github/workflows/test.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: 3.9
+          python-version: 3.11
```

Comment:
Just curious, what are the `3.9` to `3.11` changes for? Will `0.34.0` not support `python 3.9`? Should not our base tests be against the lowest version which will ensure they work in higher versions due to backwards compatibility?

Reply:
I thought 3.11 because of speed, we test 3.9 already in the matrix.

But I suppose it is not in the scope of this PR, so I reverted.

3.9 of course is still supported.